components/object-property-list/object-property-list-item.js

```diff
@@ -4,12 +4,13 @@ import { css, html, LitElement, nothing } from 'lit';
 import { getSeparator } from '@brightspace-ui/intl/lib/list.js';
 import { offscreenStyles } from '../offscreen/offscreen.js';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
 
 /**
  * A single object property, to be used within an object-property-list,
  * with an optional icon.
  */
-export class ObjectPropertyListItem extends RtlMixin(LitElement) {
+export class ObjectPropertyListItem extends SkeletonMixin(RtlMixin(LitElement)) {
```

Comment:
Question for discussion: in cases where skeletons are used, do we feel as though consumers know that there will be X number of items in the property list, or might they not know how many they have until their async data has loaded?

If they do know ahead of time, then I think having the items support skeletons makes sense -- you'll get nice skeletons with the dot separators between them. But if they don't know how many, then it may make sense to support skeleton on the list itself... or alternatively I guess they could render some placeholder items that may or may not represent the final real number.

Reply:
Based on my understanding from [this](http://proddevutil.desire2learn.d2l/d2c/content/product%20design/design.d2l%20archive/Skeleton%20Page%20Loaders%20%E2%80%93%20Daylight%20Design%20System.html), for lists we would want to display a potentially hard coded number of elements. But this link mostly refers to tables, not this component.

So I would assume that if we do not know how many then we could display 3 (or something like that)

Reply:
Yeah I was thinking about that too, I think both use cases could make sense.

Reply:
If skeletizing the object-property-list itself, I'd assume just one item with no separators as the safest default.